src/pages/SocialGraph.vue

```diff
@@ -56,38 +66,70 @@
       Next &gt;&gt;
     </button>
     <table>
-      <tr>
-        <th>Account</th>
-        <th>Total Count</th>
-        <th>Collections</th>
-        <th>Total Value</th>
-      </tr>
-      <tr
-        v-for="c in currentPageData"
-        :key="c.account"
-      >
-        <td>
-          <UserLink
-            :wallet="c.account"
-          />
-        </td>
-        <td>{{ c.count }}</td>
-        <table>
+      <thead>
+        <tr>
+          <th rowspan="2">
+            Account
+          </th>
+          <th colspan="3">
+            Class
+          </th>
+          <th colspan="2">
+            Collection
```

Comment:
Good use of layered header
Just call this `Total`? `Collection` sounds like there exists a concept `Collection of Classes` but there really isn't